src/lambda/__tests__/fixtures/Lambda/LambdaFunctionThatReturnsNativeString-fixture.js

```diff
@@ -16,6 +16,7 @@ export default class LambdaFunctionThatReturnsNativeString {
     },
     service: {
       provider: {
+        arquitecture: "arm64",
```

Comment:
should this be architecture (spelling)?

Reply:
fixed typo

Reply:
It's still `arquitecture`, instead of `architecture`